app-localize-behavior.html

```diff
@@ -52,6 +52,33 @@
 *        &lt;/script>
 *     </dom-module>
 *
+*
+* If the resources stored in your external file are for a single language and
+* so are not nested inside any language keys, you can pass an optional
+* `language` parameter to store the fetched resources inside that key.
+* This complements the optional third parameter, `merge`, nicely: If you pass
+* `merge = true`, the fetched resources will be merged into any existing
+* resources rather than clobbering them. Object.assign is used for the merge,
+* but you can also pass your own merge function to use that instead.
```

Comment:
"You can also pass your own merge function" -- this is no longer true, right?

Reply:
Ohhhh i see. No, this is bad news bears (and clojure doesn't like it when one thing can have 2 types). I would recommend not giving people this option, and only having merge be boolean